### ISSUE TITLE
Richard Vranch

### ISSUE THREAD

**Issue by rchurchley** [rchurchley/erdos-bacon-sabbath#12]
<a href="http://en.wikipedia.org/wiki/Richard_Vranch">Richard Vranch</a> is a comedian, writer, actor, and voiceoverer. He is part of the Comedy Store Players, a London improv comedy group, and has recently been doing stand-up comedy in French. Although he does not work as a musician, many people know him from his musical accompaniment on the original UK series <em>Whose Line is it Anyway?</em> He has a Ph.D. in physics from the University of Cambridge.

<h2>Erdős 5</h2>

<ul>
<li>Richard Vranch

<a href="http://link.aip.org/link/doi/10.1063/1.99192">Spin‐dependent recombination in irradiated Si/SiO2 device structures</a></p></li>
<li><p>M. Pepper

<a href="http://dx.doi.org/10.1021/nl903314j">Direct Observation of Nonequilibrium Spin Population in Quasi-One-Dimensional Nanostructures</a></p></li>
<li><p>D.A. Ritchie

<a href="http://dx.doi.org/10.1049/el:20064168">Spectral engineering of terahertz quantum cascade lasers using focused ion beam etched photonic lattices</a></p></li>
<li><p>Sourav Chakraborty

<a href="http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.205.3833">Property testing of equivalence under a permutation group action</a></p></li>
<li><p>László Babai

<a href="http://dx.doi.org/10.1137/0209047">Random Graph Isomorphism</a></p></li>
<li><p>Paul Erdős</p></li>
</ul>

<h2>Bacon 2</h2>

<ul>
<li><p>Richard Vranch

<em><a href="http://www.imdb.com/title/tt1956676/">The Sweeney: The Life and Work of Jim Sweeney</a></em></p></li>
<li><p>Michael McShane

<em><a href="http://www.imdb.com/title/tt0112453/">Balto</a></em></p></li>
<li><p>Kevin Bacon</p></li>
</ul>

<h2>Sabbath 2</h2>

<ul>
<li><p>Richard Vranch

<em><a href="http://en.wikipedia.org/wiki/Jammin'">Jammin’</a></em> <a href="http://epguides.com/Jammin/">(S3E2)</a></p></li>
<li><p>Herbie Flowers

<em><a href="http://www.procolharum.com/99/bh_funkist-cd.htm">Funkist</a></em></p></li>
<li><p>Tony Iommi</p></li>
</ul>

<h3>Credits</h3>

<p><em>Discovery:</em> <a href="https://twitter.com/#!/moonminerva">Sarah Falk</a>, <a href="http://richardvranch.com/">Richard Vranch</a>
